libs/coin-framework/src/api/types.ts

```diff
@@ -31,6 +31,16 @@ export type Asset = {
   native: bigint;
 };
 
+export type Intent = {
+  sender: string;
+  recipient: string;
+  amount: bigint;
+  contractAddress?: string;
+  standard?: "trc10" | "trc20";
+};
+
+export type Fees = { [feeOption: string]: bigint };
```

Comment:
```suggestion
type FeeOption = "standard" | "gross";
export type Fees = Partial<Record<FeeOption, bigint>>;
```

Ideally we would like to control the type of keys.

Reply:
Fee option will be scoped for each coin module

I don't think we want to list keys for now